src/btree/bt_sync.c

```diff
@@ -137,6 +137,57 @@ err:	WT_TRET(__wt_page_release(session, next, flags));
 }
 
 /*
+ * __sync_checkpoint_progress --
+ * 	Output a checkpoint progress message.
+ */
+static void
+ __sync_checkpoint_progress(WT_SESSION_IMPL *session)
+{
+	struct timespec cur_time;
+	WT_CONNECTION_IMPL *conn;
+	uint64_t time_diff;
+
+	conn = S2C(session);
+	time_diff = 0;
```

Comment:
Don't need to initialise to zero, we overwrite that value couple of lines below.

Reply:
Good point.